app/static/js/events/wizard/vue-app.js

```diff
@@ -1,4 +1,6 @@
 var steps = ['', 'sponsors', 'sessions-tracks-rooms'];
+var modifications = {'sessions-tracks-rooms': {sessionCreate:0, sessionRemove:0, trackCreate:0, trackRemove:0, roomCreate:0, roomRemove:0},
```

Comment:
Instead of doing all this, let the wizard itself be dynamic depending the permission a user has. For example, If a user cannot edit tracks, tracks section without any option to edit can be shown.